src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -181,10 +191,13 @@ public boolean isMegaTag1Good(int limelightNumber) {
   public boolean isMegaTag2Good(int limelightNumber) {
```

Comment:
isMegaTag1Good and this have duplicated logic, so I would condense them into one function. Also, didn't you want to do something to determine if MT2 was valid based on how similar it was to the MT1 reading? I thought that was a really good idea.

Reply:
the logic for that exists with the discrepancy stuff. It still exists, but isn't implemented in the updatePoseEstimate bc I haven't had time to test it yet.

Reply:
pushed function condensation. check it out